### PR TITLE
add git to wrapper dependencies

### PR DESCRIPTION
**Pull request by ryantm**

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -18,6 +18,7 @@ python3.pkgs.buildPythonApplication rec {
   '';
   makeWrapperArgs = [
     "--prefix PATH" ":" "${nix}/bin"
+    "--prefix PATH" ":" "${git}/bin"
```
Comment (Mic92) [inline default.nix:21]:
```suggestion
    --prefix PATH : ${stdenv.lib.makeBinPath [ nix git ]}
```

File: default.nix [hunk 2]
```diff
@@ -17,7 +17,7 @@ python3.pkgs.buildPythonApplication rec {
     mypy --strict nixpkgs_review
   '';
   makeWrapperArgs = [
-    "--prefix PATH" ":" "${nix}/bin"
+    --prefix PATH : ${stdenv.lib.makeBinPath [ nix git ]}
```
Comment (Mic92) [inline default.nix:20]: Sorry. I forget the quotes:

```suggestion
    "--prefix" "PATH" ":" "${stdenv.lib.makeBinPath [ nix git ]}"
```

**Comment by ryantm** [comment]
@Mic92 fixed and squashed.

**Comment by ryantm** [comment]
Oh, maybe I'm not using enough quotes. Why do you use so many?

**Comment by Mic92** [comment]
> Oh, maybe I'm not using enough quotes. Why do you use so many?

I was not sure how it would be passed to make wrapper but apparently not quoted. Otherwise it would just receive one but argument.

**Pull request merged**